date.h

```diff
@@ -3984,13 +3994,15 @@ format(const std::locale& loc, std::basic_string<CharT, Traits> fmt,
             if (command && !modified && ratio_less<typename Duration::period,
                                                    ratio<1>>::value)
             {
-                basic_ostringstream<CharT, Traits> os;
-                os.imbue(loc);
-                os << make_time(tp - floor<seconds>(tp));
-                auto s = os.str();
-                s.erase(0, 8);
-                fmt.insert(i+1, s);
-                i += s.size() - 1;
+                using subsecput_type = time_subsec_put<typename Duration::period>;
+                if(subsecput_type::admits_subsecs) {
+                  // skip over the T or S
+                  ++i;
+                  Duration subsecs=tp - floor<seconds>(tp);
```

Comment:
This isn't compiling for the case that `Duration` is coarser than seconds (e.g. minutes).